Source/WebCore/css/values/primitives/CSSPrimitiveNumericRange.h

```diff
@@ -61,41 +72,51 @@ struct Range {
 
     double min { -infinity };
     double max {  infinity };
-    RangeOptions options { RangeOptions::Default };
+    RangeClampOptions clampOptions { RangeClampOptions::Default };
+    RangeZoomOptions zoomOptions { RangeZoomOptions::Default };
```

Comment:
since zoom behavior is embedded into Range semantics and all callers seem to read RangeZoomOptions state.

how do you envision the feature flag in this new architecture?

Reply:
i was thinking about `constexpr RangeZoomOptions zoomOptions() const {}` and reading would be `R.zoomOptions()` which would unroll with the runtime flag logic but the issue here would be `R` accessing the context i guess? i.e BuilderState or RenderStyle?

Reply:
Yes, you need to pass the runtime options in somewhere.